sql/privilege.go

```diff
@@ -182,9 +182,10 @@ func (p *PrivilegeDescriptor) Revoke(user string, privList privilege.List) {
 }
 
 // Validate is called when writing a descriptor.
```

Comment:
You mean remove the "Validate is called when writing a descriptor"? I'd like to keep that in, it adds context about having an ID. Clarified to "database or table descriptor" though.